store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/FederatedStore.java

```diff
@@ -334,10 +350,23 @@ public Schema getSchema(final Map<String, String> config, final User user) {
      *
      * @param user        the users scope to get graphs for.
      * @param graphIdsCsv the csv of graphIds to get, null returns all graphs.
+     * @param operation   the requesting operation.
      * @return the graph collection.
      */
-    public Collection<Graph> getGraphs(final User user, final String graphIdsCsv) {
```

Comment:
I don't think it would or should be used anywhere.
The graphIdsCsv is optional value from operations during federation, ignoring this will ignore the users request to scope the operation.
Removing the the Operation parameter will remove the attempt to mitigate infinite loops, making this ticket change potentially useless.